src/components/home-components/search-bar/search-option/SearchOptionContainer.tsx

```diff
@@ -1,37 +1,46 @@
-import React, { RefObject } from 'react';
-import { css } from '@emotion/react';
+import React, { useCallback } from 'react';
 import { selectedGovernmentLocationAtoms, selectedSectorAtoms } from '../../../../recoil/atoms';
 import SearchOption from './SearchOption';
 import SearchService from '../../../../service/SearchService';
+import styled from '@emotion/styled';
+import { Button as AntdButton } from 'antd';
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  gap: 16px;
+  justify-self: center;
+  width: 70%;
+  max-width: 500px;
+`;
+
+const Button = styled(AntdButton)`
+  width: 100%;
+`;
 
 type Props = {
-  inputElement: RefObject<HTMLInputElement>;
+  searchText: string;
 };
 
-function SearchOptionContainer({ inputElement }: Props) {
+function SearchOptionContainer({ searchText }: Props) {
   const searchCompany = SearchService();
 
+  const onSearch = useCallback(() => {
+    if (!searchText || searchText === '' || !searchCompany) {
```

Comment:
```suggestion
    if (!searchText || !searchCompany) {
```
이렇게 작성해도 똑같지 않나요?? 해당 조건이 필요한 이유가 있나요?

Reply:
아 방금 확인해보니 searchText가 없어도 조회 가능하게 해야해서 `!searchText`도 삭제해주셔야할 것 같아요!

Reply:
안내 감사합니다! @Ji-InPark 님의 의도를 모르는 상태에서 리팩토링을 진행하다보니 생긴 이슈인 것 같네요 ㅠ
바로 수정해두겠습니다!